x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -174,7 +174,7 @@ export const showView: (state: EndpointState) => EndpointIndexUIQueryParams['sho
  * Returns the Host Status which is connected the fleet agent
  */
 export const hostStatusInfo: (state: Immutable<EndpointState>) => HostStatus = createSelector(
-  (state) => state.hostStatus,
+  (state: Immutable<EndpointState>) => state.hostStatus,
```

Comment:
Why was this type necessary, doesn't TS infer it from the parent scope?

Reply:
Not anymore, sadly. Reselect changed some internal typings so it no longer infers this correctly, requiring us to be explicit.

Reply:
I see.